### DISCUSSION TITLE
Download maps for offline usage

### DISCUSSION THREAD

**Discussion by deregtervictor** [OSTA-group/osta#48] [Ideas]
When downloading an area we don't download the area of the map yet. It would be nice to also save the map tiles and be able to use everything offline for your trip.

**Comment by BauwenDR** [comment]
We currently have a tile layer for displaying offline maps using the https://github.com/allartk/leaflet.offline leaflet plugin.
The map tiles do not yet get stored in the database. The problem with this plugin is that it is not made with react in mind.

If you have any ideas of how to get this plugin working (or you want to completely redo offline maps), let us know.

(the leaflet offline plugin has an open issue for a react example, so you can kill two birds with one stone)
